composer.json

```diff
@@ -16,6 +16,7 @@
         "ext-ctype": "*",
         "ext-curl": "*",
         "ext-dom": "*",
+        "ext-fileinfo": "*",
```

Comment:
Please add the same to `lib/internal/Magento/Framework/composer.json`. Otherwise this extension won't be taken into account during composer-based deployment.

Reply:
Added `"ext-fileinfo": "*",` to `lib/internal/Magento/Framework/composer.json`